projects/multiselect/src/lib/multiselect.component.ts

```diff
@@ -252,4 +252,14 @@ export class NgxMultiselectComponent extends NgxMultiselectBaseComponent {
     this._selectedOptions = selected;
     this.onChange(selected);
   }
+
+  deSelectOptions(event){
+    this._selectedOptions.ticked = !this._selectedOptions.ticked;
+    let val = this._selectedOptions && this._selectedOptions.id;
+    let changedOptions = this.getOptions().map(o => ({...o, ticked: o.id == val && this._selectedOptions.ticked}));
```

Comment:
It suppose to clear the selection, so we can consider setting `false` for all `ticked` properties

```let changedOptions = this.getOptions().map(o => ({...o, ticked: false}));```